test-suite/Makefile

```diff
@@ -174,8 +198,15 @@ all:
 	@echo ""
 	@false
 else
+ifeq ($(TIMED),)
 all: run
 	$(MAKE) report
+else
+all:
+	$(MAKE) run | tee time-of-build.log
+	python ../tools/make-one-time-file.py --real time-of-build.log
```

Comment:
@ejgallego what's the right way to get the path of make-one-time-file in dune?
It's supposed to be installed somewhere in coq-core

Reply:
You want to get the path to the build tree or the install tree?

Is this file decalred as a dependency of the rule?

Reply:
Indeed, it is not, that's the problem.

Reply:
It's included as part of coq-core in the install tree
(it's used by coq_makefile / timing tests)

Reply:
IMHO add the dependency to dune so the file is pulled from the source tree instead of the installed tree.

The source tree is here anyways.

Reply:
There are few libraries that allow you to query for installed files, but IMHO we don't need to be so fancy here.